transitmatters/t-performance-dash
# feat(trip-explorer): speed for station pairs

## Motivation

Closes https://github.com/transitmatters/t-performance-dash/issues/803



## Changes
- Add speed graph on trip explorer


![image](https://github.com/transitmatters/t-performance-dash/assets/46619169/ccc5e3e9-9e2c-4907-b168-2638fedba5f2)




## Testing Instructions
- Navigate to trip explorer page for all lines
- Verify that the new speed graph is populated correctly
